packages/api/database/migrations/1612300913780_device_token_schema.js

```diff
@@ -0,0 +1,24 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+class DeviceTokenSchema extends Schema {
+	up() {
+		this.create('device_tokens', (table) => {
+			table.increments();
+			table
+				.integer('user_id')
+				.unsigned()
+				.references('id')
+				.inTable('users');
```

Comment:
É interessante apagarmos os tokens ao excluirmos o usuário
```suggestion
				.inTable('users')
				.onUpdate('cascade')
				.onDelete('cascade');
```